<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="Auxiliar2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>310</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>127</red>
          <green>9</green>
          <blue>121</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>127</red>
          <green>9</green>
          <blue>121</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled/>
     </palette>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="Auxiliar1">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>70</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>127</red>
          <green>9</green>
          <blue>121</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>127</red>
          <green>9</green>
          <blue>121</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled/>
     </palette>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QDial" name="Entrada">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>91</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="Salida">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>180</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="dial_Aux1Out">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>50</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="dial_Aux2Out">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>290</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>200</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Habilitar</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>440</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Habilitar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_PrincipalLevel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NivelPrincipal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Aux1Level">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>80</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>NivelAux1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Aux2Level">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>320</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>NivelAux2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLoadState">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_OutRate">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Caudal salida</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_InRate">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Caudal entrada</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSaveState">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_CisternaInLph">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Aux1OutLph">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>140</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Aux2OutLph">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>380</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_CisternaOutLph">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>280</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ApplyCaps">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>470</y>
      <width>221</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Aplicar cambios de Capacidad</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ApplyFlows">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>430</y>
      <width>221</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Aplicar cambios de Caudal</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_CapPrincipal">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_CapAux2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>270</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_CapAux1">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="Principal">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>191</width>
      <height>171</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>127</red>
          <green>9</green>
          <blue>121</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>127</red>
          <green>9</green>
          <blue>121</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled/>
     </palette>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1039</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
